lib/dfe/reference_data/itt/incentives.rb

```diff
@@ -451,20 +451,22 @@ module ITT
             scholarship: '17000',
             notes: 'geography 2020-2021'
           },
+          # rubocop:disable Lint/DuplicateHashKey
```

Comment:
Why did you disable this cop?

Reply:
see below, @lewisnix21 wanted 2 identical ids...

Reply:
In Ruby, hash keys must be unique within the same hash, so defining the same key `95eebd6c-69c5-4883-b3ff-c703b559bc4b` twice will result in only the last occurrence being retained.

https://docs.ruby-lang.org/en/3.2/Hash.html

Reply:
@goodviber okay, after reviewing how other scenarios like this are handled in those two files (i.e. other groups of subjects with the same incentive notes), to be consistent and not introduce any duplicate hashing I propose we go with this:

1. Revert the change in `incentives.rb` so we use ID 80464bd0-f509-4d54-b70a-add606b1844f again
2. Where the ID 95eebd6c-69c5-4883-b3ff-c703b559bc4b appears for 2020-2021 Biology and 2020-2021 Applied Biology in `subjects.rb`, replace with the ID 80464bd0-f509-4d54-b70a-add606b1844f. Note the ID 95eebd6c-69c5-4883-b3ff-c703b559bc4b should remain for 2020-2021 Classics
3. Keep the current change to `subjects.rb` where you have replaced the ID for 2023-2024 Applied Biology from 80464bd0-f509-4d54-b70a-add606b1844f to 71730a26-b80b-440f-96ac-8c5814daa2c5

Reply:
Ok Kokey...